lib/lists/sites-list.js

```diff
@@ -33,15 +33,30 @@ function constructMediaPath(site) {
  * @returns {Array}
  */
 function generateSitesIndexOps(acc, site) {
-  var mediaDir = path.resolve(process.cwd(), site.assetDir, 'media', 'sites', site.slug),
-    mediaDirFiles = files.getFiles(mediaDir),
+  var siteId = site.subsiteSlug || site.slug,
+    mediaDir,
+    mediaDirFiles,
+    icons;
+
+  if (site.subsiteSlug) {
+    // check first in subsite directory ie public/media/sites/parent/subsite
+    mediaDir = path.resolve(process.cwd(), site.assetDir, 'media', 'sites', site.subsiteSlug);
+    mediaDirFiles = files.getFiles(mediaDir);
+    icons = _.intersection(acceptedIcons, mediaDirFiles); // https://lodash.com/docs/4.17.2#intersection
+  }
+
+  // if its not a subsite, or subsite didn't have its own icon, check parent site
+  if (!icons || icons.length == 0) {
```

Comment:
Remember to always compare with triple equals
```suggestion
  if (!icons || icons.length === 0) {
```